setup.py

```diff
@@ -102,7 +102,7 @@ def run(self):
     packages=find_packages("src"),
     zip_safe=False,
     install_requires=install_requires,
-    extras_require={"dev": dev_requires},
+    extras_require={"dev": dev_requires, "rabbitmq": ["amqp==2.1.4", "librabbitmq==1.6.1"]},
```

Comment:
We should also be making relevant changes to our Dockerfiles, right?

Reply:
Yeah, but I'm doing a bunch of local testing at the moment, which is turning up that we likely can't use `librabbitmq`